### ISSUE TITLE
Add setting for the time format

### ISSUE THREAD

**Issue by Mingun** [JetpackDuba/Gitnuro#133]
**Is your feature request related to a problem? Please describe.**
I prefer to see exact times instead of relative ones. Currently the precise date of the commit can be viewed only in a tooltip in a details commit pane (bottom-right corner). It is impossible to see the exact time in the graph.

**Describe the solution you'd like**
Add a setting with list of predefined formats or, maybe, a field with format string that will determine how to format dates.
Add tooltips to the graph when mouse over date field:

![Gitnuro_001](https://github.com/JetpackDuba/Gitnuro/assets/450131/f7cd8d3e-21d8-473f-a8df-87de33d965be)

**Describe alternatives you've considered**
I think, there is no alternatives

**Comment by dasyad00** [comment]
If this hasn't been worked on yet, I can submit a PR for this request, but I do have a couple questions:
- Is the date/time format setting going to apply for the whole app, or just the commit history view, or should there be a setting for each date/time field?
- Is the list of predefined options just "Relative" (current) and "Absolute" (date/time)? Alternatively, should the app just support multiple date time formats (without relative, or both?)?

**Comment by JetpackDuba** [comment]
I still have to think about the UI/UX, this has been the main reason for delaying this issue to v1.5.0.

As I'd like to give multiple options to customize how date is formatted but I'm not fully sure about the details.

**Comment by dasyad00** [comment]
![image](https://github.com/user-attachments/assets/898a74f0-e67b-4e2a-bc58-b5711e84a66f)

Is this along the lines of what you have in mind?

**Comment by Mingun** [comment]
I think, that the ability to specify custom format in addition to shortcuts also would be great. It also would be useful to have preview field with the current date/time.

**Comment by JetpackDuba** [comment]
> Is this along the lines of what you have in mind?

Not really, I'd like to give options to customize it in different time formats. By default it tries to use the OS date time format but the user should be able to specify a different one. 

Let's say I have my PC in English (USA), by default the format would be `MM/DD/YYYY` but I wanted to use the date as `DD/MM/YYYY`, Gitnuro should provide the option to change it.

I have a bunch of requirements in mind but nothing clear yet about how it should look.



> I think, that the ability to specify custom format in addition to shortcuts also would be great. It also would be useful to have preview field with the current date/time.

Yes, a preview should be displayed to make it easier to understand what happens after every change.
